scalableminds/webknossos
# Isosurface picking

### URL of deployed dev instance (used for testing):
- https://isohittest.webknossos.xyz

### Steps to test:
- open a segmentation in view mode
- shift click on a cell to render an isosurface
- shift click on the isosurface in the 3d viewport
- the position should jump to the clicked position
- also, there should be hovering "effects" when hovering over isosurfaces

### Issues:
- fixes #3704 

------
